scalableminds/webknossos
# Add sharing modal for view mode

This PR adds a modal to share a view only tracing accessible via the dropdown menu. If the tracing is private a token is included in the link.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a private dataset in view mode, open the modal and copy the link. Now open the link in another browser, where you are not signed in. The dataset should load without any error.
- Do the same for a public dataset. In this case no token shall be included in the sharing URL.

### Issues:
- fixes #4477

------
